boards/nxp/frdm_mcxn947/board.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024  NXP
+ * Copyright 2025 NXP
```

Comment:
should not remove 2024, ie 2024-2025

Reply:
oops, dumb mistake.  Updated now, thanks for catching it @decsny